DevHome.sln

```diff
@@ -134,6 +134,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevHome.Customization", "to
 		{54082587-A435-423F-AE1B-01B906FFA7C5} = {54082587-A435-423F-AE1B-01B906FFA7C5}
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "WindowsSandboxExtension", "WindowsSandboxExtension", "{4ACF917D-B2CC-4CF2-8EE1-0EBBB52A69F0}"

```

Comment:
The entire WindowsSandboxExtension directory should live inside the "extensions" directory.

Reply:
I was just mirroring the Hyper-V extension. Is there a reason Hyper-V does not live there?

Reply:
It's supposed to move. #2392